js/plates-view/plate-mesh.ts

```diff
@@ -434,14 +444,19 @@ export default class PlateMesh {
     const color = this.fieldColor(field);
     this.geoAttributes.color.setXYZW(id, color.r, color.g, color.b, color.a);
 
-    const colormap = this.store.colormap;
-    if (colormap === "topo" || colormap === "plate") {
-      this.geoAttributes.colormapValue.setX(id, normalizeElevation(field.elevation));
-    } else if (colormap === "age") {
-      this.geoAttributes.colormapValue.setX(id, field.normalizedAge / MAX_NORMALIZED_AGE);
-    } else if (colormap === "rock") {
-      // colormapValue will be used to pick the texture in plate-mesh-fragment.glsl.
-      this.geoAttributes.patternIdx.setX(id, ROCK_PATTERN_IDX[field.rockType] || 0);
+    if (color === USE_COLORMAP_COLOR) {
```

Comment:
I've added this line to make the flow a bit clearer / more understandable (and we will skip some operations that were not necessary).